apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowKey.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// The key of a row.
+    /// </summary>
+    public sealed class RowKey : IEquatable<RowKey>
```

Comment:
Are row keys ever compared directly? If so, it would be useful for this to implement `IComparable<RowKey>`

Reply:
I assume so. Added `IComparable` and `IComparable<RowKey>` and added a TODO to verify this assumption.